### PR TITLE
Uploaded Class 2 Homework

### PR DESCRIPTION
**Pull request by fashed**

### CODE REVIEW

File: jennifer_choi/class2/my_calc.rb [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+	puts "The answer is #{add(num1, num2)}."
+elsif operation == "subtract"
+	puts "The answer is #{subtract(num1, num2)}."
+elsif operation == "multiply"
+	puts "The answer is #{multiply(num1, num2)}."
+elsif operation == "divide"
+	puts "The answer is #{divide(num1, num2)}."
+elsif operation == "modular_division"
+	puts "The answer is #{divide(num1, num2)}."
+else
```
Comment (trivett) [inline jennifer_choi/class2/my_calc.rb]: Good work!

**Comment by trivett** [comment]
Well done, @fashed 

How's the classing going for you so far?

**Comment by fashed** [comment]
@trivett It's great, you explain the concepts well!

**Comment by trivett** [comment]
Glad to hear. By all means let me know if anything is less than clear.

**Pull request merged**